clients/imodels-client-authoring/README.md

```diff
@@ -8,7 +8,17 @@ This package contains an API client that exposes a subset of [iModels API](https
 
 ## Usage examples
 
-**Note:** Since the `@itwin/imodels-client-authoring` package extends the [`@itwin/imodels-client-management`](../imodels-client-management/README.md) package all usage examples presented for the `@itwin/imodels-client-management` package are valid for this one as well.
+Things to note:
```

Comment:
I think this should still be **Note:**

Reply:
Changed into note: https://github.com/iTwin/imodels-clients/pull/96/commits/d4639e7534694888fe37f68e3e39bfb5e74ef4ee